internal/coremsgs/en_struct_descriptions.go

```diff
@@ -385,6 +386,19 @@ var (
 	// NodeStatusDefaults field descriptions
 	NodeStatusDefaultsNamespace = ffm("NodeStatusDefaults.namespace", "The default namespace on this node")
 
+	// NodeStatusPlugins field descriptions
+	NodeStatusPluginsBlockchain    = ffm("NodeStatusPlugins.blockchain", "The blockchain plugins on this node")
+	NodeStatusPluginsDatabase      = ffm("NodeStatusPlugins.database", "The database plugins on this node")
+	NodeStatusPluginsDataExchange  = ffm("NodeStatusPlugins.dataExchange", "The data exchange plugins on this node")
+	Events                         = ffm("NodeStatusPlugins.events", "The event plugins on this node")
+	NodeStatusPluginsIdentity      = ffm("NodeStatusPlugins.identity", "The identity plugins on this node")
+	NodeStatusPluginsSharedStorage = ffm("NodeStatusPlugins.sharedStorage", "The shared storage plugins on this node")
+	NodeStatusPluginsTokens        = ffm("NodeStatusPlugins.tokens", "The token plugins on this node")
+
+	// NodeStatusPlugin field descriptions
+	NodeStatusPluginName = ffm("NodeStatusPlugin.name", "The name of the plugin")
```

Comment:
I would spell out maybe "the _configured_ name of the plugin" (trying to highlight this comes from the config file, even if just a little...)